packages/core/src/types/config/access-control.ts

```diff
@@ -89,22 +84,22 @@ type ListAccessControlObject<ListTypeInfo extends BaseListTypeInfo> = {
   // - a filter. In this case, the operation can proceed, but the filter will be additionally applied when updating/reading/deleting
   //   which may make it appear that some of the items don't exist.
   // - boolean true/false. If false, treated as a filter that never matches.
-  filter?: {
-    query?: ListFilterAccessControl<'query', ListTypeInfo>;
-    update?: ListFilterAccessControl<'update', ListTypeInfo>;
-    delete?: ListFilterAccessControl<'delete', ListTypeInfo>;
-    // create: not supported: FIXME: Add explicit check that people don't try this.
-    // FIXME: Write tests for parseAccessControl.
-  };
+  filter?:
+    | ListFilterAccessControl<FilterOperation, ListTypeInfo>
+    | {
+      query?: ListFilterAccessControl<'query', ListTypeInfo>;
+      update?: ListFilterAccessControl<'update', ListTypeInfo>;
+      delete?: ListFilterAccessControl<'delete', ListTypeInfo>;
+    };
 
   // These rules are applied to each item being operated on individually. They return `true` or `false`,
   // and if false, an access denied error will be returned for the individual operation.
-  item?: {
-    // query: not supported
-    create?: CreateListItemAccessControl<ListTypeInfo>;
-    update?: UpdateListItemAccessControl<ListTypeInfo>;
-    delete?: DeleteListItemAccessControl<ListTypeInfo>;
-  };
+  item?:
+    | {
```

Comment:
We technically support a `boolean` here

https://github.com/keystonejs/keystone/blob/c3a3789c6e4a1761a24cd02904b7549a62113363/packages/core/src/lib/core/access-control.ts#L160

I don't think we want that...